### ISSUE TITLE
"Type[StorageLevel]" has no attribute for predefined levels

### ISSUE THREAD

**Issue by sb2nov** [zero323/pyspark-stubs#414]
Following tests yield

```
[case predefinedStorageLevels]
from pyspark.storagelevel import StorageLevel 


StorageLevel.DISK_ONLY
StorageLevel.DISK_ONLY_2
StorageLevel.MEMORY_AND_DISK
StorageLevel.MEMORY_AND_DISK_2
StorageLevel.MEMORY_AND_DISK_SER
StorageLevel.MEMORY_AND_DISK_SER_2
StorageLevel.MEMORY_ONLY
StorageLevel.MEMORY_ONLY_2
StorageLevel.MEMORY_ONLY_SER
StorageLevel.MEMORY_ONLY_SER_2
StorageLevel.OFF_HEAP
[out]
```

```
"Type[StorageLevel]" has no attribute "DISK_ONLY"
"Type[StorageLevel]" has no attribute "DISK_ONLY_2"
...
```

**Comment by zero323** [comment]
That's because these member are [simply monkey patched](https://github.com/apache/spark/blob/11d3a744e20fe403dd76e18d57963b6090a7c581/python/pyspark/storagelevel.py#L52-L58) and were not picked by stubgen.

The solution proposed in #415 is good, but please suggest it as a separate PR (you can add the test data as well).

**Comment by zero323** [comment]
Resolved by #467
